### PR TITLE
Feature/im tests

### PR DESCRIPTION
**Pull request by hellais**
This add the IM tests

### CODE REVIEW

File: examples/nettests/Whatsapp.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+const options = {
+  backend: '',
+  geoipCountryPath: '',
+  geoipAsnPath: '',
+  saveRealProbeIp: false,
+  saveRealProbeAsn: true,
+  saveRealProbeCc: true,
+  noCollector: false,
+  collectorBaseUrl: '',
```
Comment (darkk) [inline examples/nettests/Whatsapp.js:11]: It's kinda confusing to see _probe-wide_ options in an example file that is related to the specific test.
Looks like layering is violated. Should "global" options be moved to separate example file and merged into `options` in run-time?

Comment (hellais) [reply examples/nettests/Whatsapp.js:11]: This is mainly just a copy paste of the WebConnectivity test. It's useful to have all the options in one place, though, so a user of the test can simply copy paste them into their source file and run it.

Maybe when we will have better docs for the node package these can be documented in some other place.

Comment (bassosimone) [reply examples/nettests/Whatsapp.js:11]: > I's useful to have all the options in one place

Makes sense

**Review by bassosimone — APPROVED** [review]
:whale:

**Pull request merged**